src/applications/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -103,18 +117,35 @@ class RoutedSavableReviewPage extends React.Component {
       user
     } = this.props;
 
+    const submitController = (
+      <SubmitController
+        formConfig={formConfig}
+        pageList={pageList}
+        path={path}
+        renderErrorMessage={this.renderErrorMessage}/>
+    );
+
+    let submitContent = submitController;
+
+    if (formConfig.downtime) {
+      submitContent = (
+        <DowntimeNotification
```

Comment:
I didn't think we'd want to put this in SubmitController, since it's very vets specific.